### PR TITLE
ceph: do not include ob/obc/volumes crds in csv

### PR DESCRIPTION
**Pull request by leseb**
**Description of your changes:**

We don't want to include those CRDs anymore, as they get imported from
another operator.
This trims further down the size of our CSV generation.

Additionally, we always clean the csv directory before proceeding.

Signed-off-by: Sébastien Han <seb@redhat.com>

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

@jarrpa FYI.

### CODE REVIEW

File: build/crds/build-crds.sh [hunk 1]
```diff
@@ -15,7 +15,6 @@
 # limitations under the License.
 
 set -o errexit
-set -o nounset
```
Comment (obnoxxx) [inline build/crds/build-crds.sh:18]: Should this be a different commit? Doesn't seem obviously related...

Comment (leseb) [reply build/crds/build-crds.sh:18]: It is because of the `-z` check and we now use unbound variables.

File: build/crds/build-crds.sh [hunk 2]
```diff
@@ -45,9 +44,12 @@ copy_ob_obc_crds() {
...
 
-generating_crds() {
-  echo "Generating crds.yaml"
+generating_crds_v1() {
+  echo "Generating v1 in crds.yaml"
   "$CONTROLLER_GEN_BIN_PATH" "$CRD_OPTIONS" paths="./pkg/apis/ceph.rook.io/v1" output:crd:artifacts:config="$OLM_CATALOG_DIR"
+}
+
+generating_crds_v1alpha2() {
   "$CONTROLLER_GEN_BIN_PATH" "$CRD_OPTIONS" paths="./pkg/apis/rook.io/v1alpha2" output:crd:artifacts:config="$OLM_CATALOG_DIR"
```
Comment (travisn) [inline build/crds/build-crds.sh:53]: Do we even need to generate the v1alpha2 anymore? If this just affects the csv, I don't think anyone is using the  flex driver in that case.

Comment (leseb) [reply build/crds/build-crds.sh:53]: This affects `crds.yaml` too.

**Review by travisn — APPROVED** [review]
